api/src/utils/mailer.ts

```diff
@@ -0,0 +1,65 @@
+import nodemailer from 'nodemailer';
+import { IUser, IPitch } from 'ssw-common';
+import { pitchStatusEnum } from './enums';
+
+type message = { to: string; from: string; subject: string; html: string };
+
+export const approvedMessage = (
+  author: IUser,
+  pitch: IPitch,
+  approver: IUser,
+): message => {
```

Comment:
Last thing I swear, instead of doing

```
const approvedMessage = (author, pitch, approve) => { 

const m = `message`

return m
```

You can just do

```
const approveMessage = (...) => (
`message`
);